src/pages/about/about.js

```diff
@@ -12,9 +12,10 @@ import Footer from "../../components/footer/footer";
  * @constructor
  */
 export default function About() {
+	const displayText = "About";
```

Comment:
This is unnecessary. No need to define a variable to pass in. Just pass the value as the parameter directly. `displayText={'ABOUT'}`

As a side note, all the titles should be in all caps. See the current production release of the site.